sdk/web-pubsub/web-pubsub-client-protobuf/src/clientProto.d.ts

```diff
@@ -0,0 +1,1665 @@
+import * as $protobuf from "protobufjs";
```

Comment:
is this generated? can we put it into a generated folder and comment it accordingly if so?

Reply:
Yes it's generated. Moved to generated folder